service/src/config/capabilityStatementResources.json

```
@@ -297,7 +297,7 @@
             }
           ],
           "name": "package",
```

```suggestion
          "name": "cqfm-package",
```
I think this one should be `cqfm-package` too.